cirq-google/cirq_google/calibration/engine_simulator_test.py

```diff
@@ -444,6 +444,15 @@ def test_from_characterizations_sqrt_iswap_when_invalid_arguments_fails():
         )
```

Comment:
I think this needs a test similar to the `test_from_dictionary_sqrt_iswap_simulates_correctly` test.

Reply:
Added a test.